esphome/components/pmsx003/sensor.py

```diff
@@ -35,11 +36,18 @@
 
 CODEOWNERS = ["@ximex"]
 DEPENDENCIES = ["uart"]
+AUTO_LOAD = ["aqi"]
 
 pmsx003_ns = cg.esphome_ns.namespace("pmsx003")
 PMSX003Component = pmsx003_ns.class_("PMSX003Component", uart.UARTDevice, cg.Component)
 PMSX003Sensor = pmsx003_ns.class_("PMSX003Sensor", sensor.Sensor)
 
+CONF_AQI = aqi.CONF_AQI
+CONF_CALCULATION_TYPE = aqi.CONF_CALCULATION_TYPE
+UNIT_INDEX = "index"
+
+AQI_CALCULATION_TYPE = aqi.AQI_CALCULATION_TYPE
+
```

Comment:
```suggestion
UNIT_INDEX = "index"
```
Can you just import CONF_AQI etc from aqi like we do for esphome.const, can remove these lines.